app/Http/Controllers/AdminController.php

```diff
@@ -11,8 +11,31 @@ class AdminController extends Controller
     public function getBasicAdminPage()
     {
         $allUsers = User::all();
+        $allUserCount = User::all()->count();
+        $user = Auth::user();
+        $canEmailCount = 0;
+        foreach($allUsers as $users){
+            if($users->can_contact === true)
+            {
+                $canEmailCount++;
+            }
+        }
+        $userAndEmail = [];
+        $userAndEmail[] = ["title" => "user", "count" => $allUserCount, "view" => route("getUsers")];
+        $userAndEmail[] = ["title" => "email", "count" => $canEmailCount, "view" => route("viewEmail")];
+        if ($user->is_admin === false)
+            abort(404);
+        else
+            return view("admin.dashboard", compact('userAndEmail', 'allUsers'));
+    }
+
+    public function viewEmail()
+    {
+        $allUsers = User::all();
+        $all = User::orderBy('id')->paginate(6);
         $user = Auth::user();
         $canEmail = array();
+        $canBeEmailed = User::where('can_contact', true)->orderBy('id')->paginate(6);
```

Comment:
For this, add an extra line of code to stop the two paginators from colliding: 

`$canBeEmailed->setPageName("contactable");`

For more info, see: 
https://laravel.com/docs/7.x/pagination
and 
https://stackoverflow.com/questions/24086269/laravel-multiple-pagination-in-one-page